core/src/main/java/org/jenkins/ui/symbol/Symbol.java

```diff
@@ -88,12 +103,37 @@ private static String loadSymbol(String namespace, String name) {
                 LOGGER.log(Level.FINE, "Failed to load symbol " + name, e);
             }
         }
-        return markup.replaceAll("(<title>).*?(</title>)", "$1$2")
-                     .replaceAll("<svg", "<svg aria-hidden=\"true\"")
-                     .replaceAll("(class=\").*?(\")", "")
-                     .replaceAll("(tooltip=\").*?(\")", "")
-                     .replaceAll("(data-html-tooltip=\").*?(\")", "")
-                     .replace("stroke:#000", "stroke:currentColor");
+        DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+        try {
+            dbf.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true);
+            DocumentBuilder db = dbf.newDocumentBuilder();
+            Document doc = db.parse(new StringInputStream(markup));
+            Element root = doc.getDocumentElement();
+            NodeList titleList = root.getElementsByTagName("title");
+            for (int i = 0; i < titleList.getLength(); i++) {
```

Comment:
As before just emptying the title element. But why not just delete them completely?